javascript-cont/destructuring.md

```diff
@@ -2,6 +2,14 @@
 
 <iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=451968fb-cee3-408a-b3c8-ade2000e6320&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
 
+### !callout-info
```

Comment:
I agree that there's some value in leaving the video in despite the drift. We could add a sentence like "The content and code examples in this lesson take precedence over the examples shown in the video".

I don't think the 'take precedence' is exactly right, but I'm trying to convey 'If you get tripped up trying to match the video to this lesson, then what we have in the lesson is preferred'.

Although, that might be overkill and I'm fine leaving such a sentence out of the call out too.

Reply:
What about something like "In this course, the style and content and code examples in this lesson are preferred over the examples shown in the video."?